### DISCUSSION TITLE
Activate auto update for Positioner

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [chakra-ui/zag#1200] [Q&A]
How to activate auto update for position or width of `Positioner` when width or height of  `Trigger` is changing?
It seems setting `{ placement: "bottom", sameWidth: true }` as `positioning` value is not working. At least it should be the same width of `Trigger` when it is changing.

Based on this page [Changing the placement](https://zagjs.com/components/react/tooltip#changing-the-placement) and this page [Options](https://floating-ui.com/docs/autoUpdate#options) adding `AutoUpdateOptions` as `listeners` is not working.

**Comment by isBatak** [comment]
It's because listeners a re always set to false https://github.com/search?q=repo%3Achakra-ui%2Fzag%20listeners%3A%20false%2C&type=code 

@segunadebayo is this a bug?

**Reply by segunadebayo** [reply]
Oh no, that's the wrong action you're looking at @isBatak.

The underlying issue is that the auto-update function doesn't consider the root parent (window). Fixed in a PR
